### PR TITLE
Updates to Serverless upgrade procedure for OCP 4.5

### PR DESCRIPTION
**Pull request by abrennan89**

### CODE REVIEW

**Comment by mgencur** [comment]
Looks good. I think we will need similar docs for 4.3 and 4.4.

File: modules/serverless-upgrade-sub-channel.adoc [hunk 1]
```diff
@@ -5,22 +5,22 @@
...
+* Finally, after you have upgraded to 1.7.0, upgrade to the latest version by selecting the `4.5` channel.
 
 [IMPORTANT]
 ====
 After each channel change, wait for the pods in the `knative-serving` namespace to get upgraded before changing the channel again.
 ====
 
 .Prerequisites
-* You have installed a Technology Preview version of {ServerlessOperatorName}, and have selected Automatic updates during the installation process.
+* You have installed a previous version of {ServerlessOperatorName}, and have selected Automatic updates during the installation process.
```
Comment (jboxman) [inline modules/serverless-upgrade-sub-channel.adoc:23]: Is _Automatic_ capitalized?

**Review by jboxman — APPROVED** [review]
@abrennan89, just one question. Otherwise LGTM.

File: modules/serverless-upgrade-sub-channel.adoc (hunk 1 above)
Comment (abrennan89) [reply modules/serverless-upgrade-sub-channel.adoc:23]: Yeah, it comes from the actual UI https://docs.openshift.com/container-platform/4.4/operators/olm-adding-operators-to-cluster.html#olm-installing-operators-from-operatorhub_olm-adding-operators-to-a-cluster :slightly_smiling_face:

**Comment by abrennan89** [comment]
Note: merge at 1.7.1 release (06/02/2020)

**Pull request merged**